contents/handbook/company/team/joe-martin-goals.md

```diff
@@ -7,17 +7,15 @@ showTitle: true
 This is a non-exhaustive list of things I'm currently trying to accomplish. It's likely that ad-hoc tasks or other priorities will appear to alter these and that's OK. For information on how I like to work, refer to [my README](https://posthog.com/handbook/company/team/joe-martin). 
 
 
-### Execution - Q4 2021
-- **Content:** Get the content pipeline working properly, with a clear content calendar. Execute against the content strategy, set the Technical Content Marketer up for success. 
-- **Landing Pages:** Work with Team Design to [get a usable landing page template made](https://github.com/PostHog/posthog.com/issues/2098) and create competitor landing pages, with further pages planned against the content strategy. Identify SEO opportunities with landing pages and hit some of the big ones.
-- **Case studies:** Get [the customer stories section](https://github.com/PostHog/posthog.com/issues/1707) of the site live and continue to build on them with [Crypto and Healthcare content](https://github.com/PostHog/posthog.com/issues/2092). Bring case study content more into the forefront across PostHog documentation and webpages. 
+### Execution - Q1 2022
+- **Community:** Organize the first PostHog community meet-up, with a good speaker. Retro the event and roll changes into a second event. Continue to attend and report back on other community events and meet-ups. 
+- **Case studies:** Secure an enterprise-level case study. Enhance the Customers section of the site and make plans to better leverage this content. 
+- **Content:** Support the Content team in creating a decent pipeline of good content that continues to deliver. 
+- **Partnerships:** Find further partnership opportunities to grow the community. 
```

Comment:
I'm curious if you could be a bit more specific about what this means - can you give an example of a previous one for context? (As 'partnerships' can be very broad)

Reply:
Ah, sorry yes that is vague - partly deliberately so, but also partly not. 

I was generally thinking of content/event marketing partnerships. The blog posts with Scarf which we did would be one example, as is having MentionMe at the HostHog. Hightouch was going to be another, but Andy has taken that on after I arranged an intro.

Reply:
```suggestion
- **Marketing Partnerships:** Find further partnership opportunities around content, community events and other marketing activities. 
```